schema.graphql

```diff
@@ -1,15 +1,15 @@
 type Event @entity {
   id: ID!
-  blockId: Int!
+  block: Block!
   eventIdx: Int!
   extrinsicIdx: Int
   specVersionId: Int!
   moduleId: String! @index(unique: false)
   eventId: String! @index(unique: false)
   attributesTxt: String!
-  eventArg_0: String
-  eventArg_1: String
-  eventArg_2: String
+  eventArg_0: String @index(unique: false)
```

Comment:
Remove. Can't index event_args with a btree which this generates

Reply:
I think this comment still stands, although I don't quite remember my reasoning from last month. 

Some events like batch can have very large inputs, but BTree indexes have a limited size.

If we want to index something like this I think it needs to be a GIN.